src/view_page.cpp

```diff
@@ -551,7 +553,26 @@ void View::DrawBracketsq(DeviceContext *dc, int x, int y1, int y2, int staffSize
 void View::DrawBrace(DeviceContext *dc, int x, int y1, int y2, int staffSize)
 {
     assert(dc);
-
+    if (m_doc->GetOptions()->m_useBraceGlyph.GetValue()) {
+        Glyph *glyph = Resources::GetGlyph(SMUFL_E000_brace);
+        if (glyph) {
+            FontInfo* font = m_doc->GetDrawingSmuflFont(staffSize, false);
+            int dummy, g_w, g_h;
+            glyph->GetBoundingBox(dummy, dummy, g_w, g_h);
+            const float ratio = static_cast<float>(glyph->GetUnitsPerEm()) / font->GetPointSize();
```

Comment:
I think you can use `Doc::GetGlyphHeight`

Reply:
I guess it's not good idea. In this case we will call `Doc::GetGlyphHeight` and `Doc::GetGlyphWidth` . Both methods call `GetBoundingBox` and do another calculations which is not necessary.